OpenSim/Common/Test/testComponentInterface.cpp

```diff
@@ -1926,6 +1928,48 @@ void testAliasesAndLabels() {
     SimTK_TEST(foo->getInput("listInput1").getLabel(1) == "thud");
 }
 
+void testSignalGenerator() {
```

Comment:
:+1: for the test case. It really doesn't belong in `testComponentInterface`. We could start a new test suite (maybe `testFunctions`) for `Function`s and some of its primary clients, e.g. `SignalGenerator` to verify their correctness.